src/ModelContextProtocol/Server/McpServerResource.cs

```diff
@@ -0,0 +1,17 @@
+using ModelContextProtocol.Protocol.Types;
+
+namespace ModelContextProtocol.Server;
+
+/// <summary>
+/// Represents a resource that the server supports.
+/// </summary>
+public class McpServerResource : IMcpServerPrimitive
```

Comment:
McpServerTool and McpServerPrompt are both abstract, with abstract members. This should be the same.